lib/absinthe/schema.ex

```diff
@@ -534,9 +534,8 @@ defmodule Absinthe.Schema do
   """
   @spec used_types(t) :: [Type.t()]
   def used_types(schema) do
```

Comment:
We should maybe rename this to `referenced_types` and deprecate this function?

Reply:
Would you want to hard-deprecate or just log a warning?

Reply:
I think we `@deprecate` it, so that it gets a compile time warning.

Reply:
done!